test/TestIntegration.java

```diff
@@ -885,6 +885,42 @@ public void compareFilters() throws Exception {
     assertEq("v3", dependent_value_rows.get(0).get(0).value());
   }
 
+    /**
+            * Simple reverse scan using reverse get of HBase 0.98+
+            * @throws Exception
+    */
+    @Test
+    public void reverseFetch() throws Exception {
+        client.setFlushInterval(FAST_FLUSH);
+
+        // All puts below should be read in scan
+        final PutRequest put1 = new PutRequest(table, "rf1", family, "qa1", "v1");
+        final PutRequest put2 = new PutRequest(table, "rf1", family, "qb2", "v2");
+        final PutRequest put3 = new PutRequest(table, "rf2", family, "qc3", "v3");
+        final PutRequest put4 = new PutRequest(table, "rf2", family, "qd4", "v4");
+        final PutRequest put5 = new PutRequest(table, "rf3", family, "qd5", "v5");
+        final PutRequest put6 = new PutRequest(table, "rf3", family, "qd6", "v6");
+
+        Deferred.group(Deferred.group(client.put(put1), client.put(put2), client.put(put5)), Deferred.group(
+                client.put(put3), client.put(put4), client.put(put6))).join();
+        final Scanner rev_scanner = client.newScanner(table);
+        rev_scanner.setFamily(family);
+        rev_scanner.setStartKey("rf3");
+        rev_scanner.setStopKey("rf1");
```

Comment:
Should we explicitly test cases to make sure that StopKey is exclusive. Also, should we have a test case where StopKey is not specified (since it's an optional parameter)?